src/components/datagrid/data_grid.spec.tsx

```diff
@@ -82,6 +82,46 @@ describe('EuiDataGrid', () => {
             .should('be.greaterThan', firstHeight);
         });
     });
+
+    it.only('accounts for a horizontal scrollbar', () => {
```

Comment:
Looks like CI is yelling about this

```suggestion
    it('accounts for a horizontal scrollbar', () => {
```